mkdocs.yml

```
@@ -45,6 +45,7 @@ nav:
           - tutorial/github/github_pages.md
   - Guides:
       - guides/index.md
+      - Virtual Environments: guides/environments.md
```

```suggestion
      - Virtual Environments: guides/environments.md
```

Put at the top because it's early in the journey - is there a logical structure you're aiming at?

---

nope not yet... just stuff I don't want to forget about